src/main/java/com/mesosphere/velocity/marathon/MarathonRecorder.java

```diff
@@ -108,21 +110,21 @@ public boolean needsToRunAfterFinalized() {
      * @param logger a build's logger
      * @param text   message to log
      */
-    private void log(final PrintStream logger, final String text) {
+    private void log(PrintStream logger, String text) {
         logger.println("[Marathon] " + text);
     }
 
     @Override
     public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListener listener)
             throws InterruptedException, IOException {
-        final boolean     buildSucceed = build.getResult() == null || build.getResult() == Result.SUCCESS;
-        final EnvVars     envVars      = build.getEnvironment(listener);
-        final PrintStream logger       = listener.getLogger();
+        boolean buildSucceed = build.getResult() == null || build.getResult() == Result.SUCCESS;
+        EnvVars envVars = build.getEnvironment(listener);
+        PrintStream logger = listener.getLogger();
```

Comment:
there are aspects of this change that I really like... the spacing removal makes sense... I'm not sure why they were like that.   It is not necessary to remove the `final`